tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,332 @@
--------------------------------
-
-- **Raw data visualization** to visualize recordings, can also use
-  *mne_browse_raw* for extended functionality (see :ref:`ch_browse`)
-- **Epoching**: Define epochs, baseline correction, handle conditions etc.
-- **Averaging** to get Evoked data
-- **Compute SSP projectors** to remove ECG and EOG artifacts
-- **Compute ICA** to remove artifacts or select latent sources.
-- **Maxwell filtering** to remove environmental noise.
-- **Boundary Element Modeling**: single and three-layer BEM model
-  creation and solution computation.
-- **Forward modeling**: BEM computation and mesh creation
-  (see :ref:`ch_forward`)
-- **Linear inverse solvers** (MNE, dSPM, sLORETA, eLORETA, LCMV, DICS)
-- **Sparse inverse solvers** (L1/L2 mixed norm MxNE, Gamma Map,
-  Time-Frequency MxNE)
-- **Connectivity estimation** in sensor and source space
-- **Visualization of sensor and source space data**
-- **Time-frequency** analysis with Morlet wavelets (induced power,
-  intertrial coherence, phase lock value) also in the source space
-- **Spectrum estimation** using multi-taper method
-- **Mixed Source Models** combining cortical and subcortical structures
-- **Dipole Fitting**
-- **Decoding** multivariate pattern analysis of M/EEG topographies
-- **Compute contrasts** between conditions, between sensors, across
-  subjects etc.
-- **Non-parametric statistics** in time, space and frequency
-  (including cluster-level)
-- **Scripting** (batch and parallel computing)
-
-What you're not supposed to do with MNE Python
-----------------------------------------------
-
-- **Brain and head surface segmentation** for use with BEM
-  models -- use Freesurfer.
-
-Installation of the required materials
----------------------------------------
-
-See :ref:`install_python_and_mne_python`.
-
-From raw data to evoked data
-----------------------------
-
-.. _ipython: https://ipython.org/
-
-Now, launch `ipython`_ (Advanced Python shell) using the QT backend, which
-is best supported across systems:
-
-.. code-block:: console
-
-    $ ipython --matplotlib=qt
-
-.. note:: In IPython, you can press **shift-enter** with a given cell
-          selected to execute it and advance to the next cell.
-
-          Also, the standard location for the MNE-sample data is
-          ``~/mne_data``. If you downloaded data and an example asks you
-          whether to download it again, make sure the data reside in the
-          examples directory and you run the script from its current directory.
-
-          From IPython e.g. say:
-
-          .. code-block:: IPython
-
-              In [1]: cd examples/preprocessing
-              In [2]: %run plot_find_ecg_artifacts.py
-
-First, load the mne package:
-"""
-
-import mne
 
-##############################################################################
-# If you'd like to turn information status messages off:
+Overview of MEG/EEG analysis with MNE-Python
+============================================
```

Comment:
Would it make sense to add a table of contents at the top since it's a long-ish tutorial?